pegasus/sites.v3/code.org/public/learn/index.redirect

```diff
@@ -1 +0,0 @@
-https://hourofcode.com/learn
```

Comment:
Ah yep, it is the main one. It's possible it's not getting much traffic, I only found three places where this is linked on the code.org site:
<img width="432" alt="Screenshot 2024-11-22 at 11 03 25 AM" src="https://github.com/user-attachments/assets/2ab8226b-6b18-4c61-aa04-6a18dd90e34a">

But should we keep this one just in case? cc @dmcavoy

Reply:
If you look at the raw data tab in the gsheet I see ~400K /learn redirects, so I think we should keep that one :). Worth a look to see if we have other /index redirects that aren't being picked up correctly by my vlookup. Apologies for the oversight!

Reply:
Added a fix to the gsheet to handle the /index suffix -- the data for the /learn redirect is now accurate! I'll alert Dani as well in Slack.

Reply:
Phew, thank you for catching this! I put this one back and will move the corresponding row in the gsheet.